modular/executor/migrate_task.go

```diff
@@ -68,7 +69,11 @@ func (e *ExecuteModular) HandleMigrateGVGTask(ctx context.Context, task coretask
 		}
 
 		for index, object := range objectList {
-			if err = e.doObjectMigration(ctx, task, bucketID, object); err != nil && !e.enableSkipFailedToMigrateObject {
+			if err = e.checkAndRenewGVGSignature(gvgTask.(*gfsptask.GfSpMigrateGVGTask)); err != nil {
```

Comment:
currently, if the `SignMigrateGVG` failed for once, then the whole migration task failed, I think we should make it more robust: is it ok that does not return error before the signature indeed expire, so it can retry to request to sign next time. 

The function name can change to `checkAndTryRenewSig`

Reply:
Fixed.